_data/garciaErick/garciaErick.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Erick Garcia
+institution: University of Texas at El Paso (UTEP)
+profile_pic: erick.jpg
+quote: I told you I was going to get my quarantine graduation MOM (╯°□°）╯︵ ┻━┻!
```

Comment:
Please remove all the special character from the quote to maintain the same format given in the [instruction](https://github.com/education/graduation#3-add-your-profile-information)